src/autosuggest/internal.tsx

```diff
@@ -139,10 +139,23 @@ const InternalAutosuggest = React.forwardRef((props: InternalAutosuggestProps, r
   };
 
   const handlePressArrowDown = () => {
+    if (autosuggestItemsState.items.length - 1 === autosuggestItemsState.highlightedIndex) {
```

Comment:
Why did you implement it here and not inside `autosuggestItemsHandlers.moveHighlightWithKeyboard`?

We have some shared dropdown code for all components. It is possible to implement it here and cover ButtonDropdown too? (currently it does not get the chage)

Reply:
Okay, got this. Sounds good by now